server.js

```diff
@@ -259,19 +259,25 @@ api.put('/documents/:uuid', bodyParser.json(), (req, res) => {
     [uuid],
     res,
     function (data) {
-      var document = {};
+      let document = {};
       document.uuid = uuid;
       document.content = req.body.content;
       document.metadata = JSON.stringify(req.body.metadata);
       document.last_modified = moment().format('YYYY-MM-DD HH:mm:ss');
-      var sql = '';
+      let sql = '';
       if (data.rows.length == 0) {
         sql = 'INSERT into resume (content, uuid, path, version, last_modified, metadata) VALUES($1, $2, $3, $4, $5, $6) RETURNING id';
       } else {
         sql = 'UPDATE resume SET content = $1, path = $3, version = $4, last_modified = $5, metadata = $6 where uuid = $2';
       }
 
-      var path = buildPath(req.body.metadata.name + '');
+      let path = '';
+
+      if(req.body.metadata.firstname){
+        path = buildPath(`${req.body.metadata.name} ${req.body.metadata.firstname}`);
+      }else {
+        path = buildPath(req.body.metadata.name);
```

Comment:
Before the change, there was a conversion to string here (using `+ ''`). I don't know why it was there, but wouldn't it be best to keep it?